formatters/lcov/lcov.go

```diff
@@ -43,13 +43,14 @@ func (r Formatter) Format() (formatters.Report, error) {
 		return rep, errors.WithStack(err)
 	}
 
+	var gitHead, _ = env.GetHead()
+
 	var sf formatters.SourceFile
 	curLine := 1
 
 	for _, line := range bytes.Split(b, []byte("\n")) {
 		if bytes.HasPrefix(line, []byte("SF:")) {
 			name := string(bytes.TrimSpace(bytes.TrimPrefix(line, []byte("SF:"))))
-			var gitHead, _ = env.GetHead()
```

Comment:
nice! this was a super oversight on my part 🤦‍♀️
i think we can probably improve this on other formatters as well.